services/security/identity/identity.go

```diff
@@ -44,6 +44,8 @@ func getIdentity(id *wssdcloudsecurity.Identity) *security.Identity {
 			CloudPort:     &id.CloudPort,
 			CloudAuthPort: &id.CloudAuthPort,
 		},
+		AutoRotate:    id.AutoRotate,
```

Comment:
Can we add a check that the loginpath is passed of autorotate is enabled? This will help up return early